awswrangler/_databases.py

```diff
@@ -60,11 +60,19 @@ def _get_connection_attributes_from_catalog(
                     f"No CA certificate found at {ssl_cert_path}."
                 )
         ssl_context = ssl.create_default_context(cadata=ssl_cadata)
-
+    print("CONNECTION DETAILS", details)
```

Comment:
To avoid leaking credentials into the logs probably would better to remove this please :)